test/CacheTest.php

```diff
@@ -41,37 +49,37 @@ protected function setUp() : void
      *
      * @return void
      */
-    protected function tearDown() : void
+    protected function tearDown(): void
     {
         if (file_exists($this->cacheFile)) {
             unlink($this->cacheFile);
         }
     }
 
-    public function testRetrievingSmdCacheShouldReturnFalseIfCacheDoesNotExist()
+    public function testRetrievingSmdCacheShouldReturnFalseIfCacheDoesNotExist(): void
```

Comment:
I don't really care if return type declared on tests. They provide no impact on code quality. I personally prefer not to have them to save me some effort when writing tests and also save extra 6 characters for lengthy verbose test names

Reply:
I get confused having different CS definitions within the same project, especially as I have test and implementation classes next to each other during development. Maybe these method names will be refactored during the coming changes.

@Xerkus Would it be okay for you to keep return type declarations or would you like them removed?

Reply:
Either way works, no established rule.  Just keep it consistent within the repo

Reply:
👍🏻